File: Data Structures & Algorithms/two-integer-sum-ii/submission-4.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        int n = numbers.size();
        for(int left = 0; left < n-1; left++){
            for(int right = left+1; right < n; right++){
                if((target - numbers[right]) == numbers[left]){
                    return {left+1, right+1};
                }
            }
        }
        return {0,0};
    }
};

File: Data Structures & Algorithms/binary-search/submission-4.py
class Solution:
    def search(self, nums: List[int], target: int) -> int:
        p1, p2 = 0, len(nums) - 1
        while p1 <= p2:
            mid = (p1 + p2) // 2
            if nums[mid] == target:
                return mid
            elif nums[mid] > target:
                p2 = mid - 1
            else:
                p1 = mid + 1
        return -1